exchanges/huobi/huobi.go

```diff
@@ -64,6 +67,49 @@ type HUOBI struct {
 	AccountID string
 }
 
+// GetMarginRates gets margin rates
+func (h *HUOBI) GetMarginRates(symbol string) (MarginRatesData, error) {
+	vals := url.Values{}
+	if symbol != "" {
+		vals.Set("symbols", symbol)
+	}
+	var resp MarginRatesData
+	return resp, h.SendAuthenticatedHTTPRequest(http.MethodGet, huobiMarginRates, vals, nil, &resp, false)
+
+}
+
+// GetSwapFundingRates gets funding rates data
+func (h *HUOBI) GetSwapFundingRates(contract string) (FundingRatesData, error) {
+	vals := url.Values{}
+	vals.Set("contract_code", contract)
+	type response struct {
+		Response
+		Data FundingRatesData `json:"data"`
+	}
+	var result response
+	err := h.SendHTTPRequest(common.EncodeURLValues("https://api.hbdm.com/"+huobiSwapFunding, vals), &result)
+	if result.ErrorMessage != "" {
+		return FundingRatesData{}, errors.New(result.ErrorMessage)
+	}
+	return result.Data, err
+}
+
+// GetSwapMarkets gets data of swap markets
+func (h *HUOBI) GetSwapMarkets(contract string) ([]SwapMarketsData, error) {
+	vals := url.Values{}
+	vals.Set("contract_code", contract)
+	type response struct {
+		Response
+		Data []SwapMarketsData `json:"data"`
+	}
+	var result response
+	err := h.SendHTTPRequest(common.EncodeURLValues("https://api.hbdm.com/"+huobiSwapMarkets, vals), &result)
```

Comment:
can move the url out into its own const string to follow usual syntax and not include as part of encodeurlvalues